src/index.html

```diff
@@ -13,6 +13,131 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div
+        class="calendar__day"
+        data-day="1"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="2"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="3"
+      ></div>
+      <div
+        class="calendar__day"
+        data-day="4"
+      ></div>
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-sun calendar--month-length-31">
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
No need to count days. It's enough to have classes with modifiers on the calendar and regular class here